app/src/main/res/layout/image_card_view.xml

```diff
@@ -28,7 +27,8 @@
             lb:layout_viewType="main"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:scaleType="centerCrop"
+            android:scaleType="fitXY"
```

Comment:
This should be centerCrop for images that are resized, mostly noticeable on image libraries.